pandas/tests/typing/valid/test_frame.py

```diff
@@ -35,17 +35,17 @@ def test_types_to_csv() -> None:
     # variable has type "str")
     csv_df: str = df.to_csv()  # type: ignore[assignment]
 
-    with tempfile.NamedTemporaryFile() as file:
+    with tempfile.Namedtempfile.TemporaryFile() as file:
```

Comment:
```py
[gw1] linux -- Python 3.8.12 /home/circleci/miniconda3/envs/pandas-dev/bin/python

    def test_types_to_csv() -> None:
        df = pd.DataFrame(data={"col1": [1, 2], "col2": [3, 4]})
        # error: Incompatible types in assignment (expression has type "Optional[str]",
        # variable has type "str")
        csv_df: str = df.to_csv()  # type: ignore[assignment]
    
>       with tempfile.Namedtempfile.TemporaryFile() as file:
E       AttributeError: module 'tempfile' has no attribute 'Namedtempfile
```


I assume you mean either `tempfile.TemporaryFile()` or `tempfile.NamedTemporaryFile()`